libcxx/include/__exception/exception_ptr.h

```diff
@@ -29,22 +29,21 @@
 
 namespace __cxxabiv1 {
 
+#    if defined(__wasm__)
+typedef void* (*__libcpp_exception_destructor_func)(void*);
```

Comment:
Could keep the existing comment here `// In Wasm, a destructor returns its argument` to explain why it is different?